addons/mrp/views/stock_move_views.xml

```diff
@@ -79,7 +79,7 @@
                                 <span><field name="product_uom_qty" readonly="1" nolabel="1"/></span>
                                 <span><field name="product_uom_id" attrs="{'readonly': [('id', '!=', False)]}" nolabel="1"/></span>
                             </div>
-                            <field name="lot_id" string="Lot/Serial number" groups="stock.group_production_lot" readonly="1" attrs="{'invisible': [('lots_visible', '=', False)]}"/>
+                            <field name="lot_id" string="Lot/Serial number" groups="stock.group_production_lot" readonly="1"/>
```

Comment:
`lots_visible` still exists on the model ; invisible isn't needed anymore?

Reply:
Add <field name="lots_visible" invisible="1"/> instead

Reply:
Removing the `attrs` keeps the behavior as is. We can also add the field `lots_visible` in the view...